packages/toast/src/InternalToast/InternalToast.styles.ts

```diff
@@ -84,7 +84,7 @@ export const toastThemeStyles: Record<Theme, string> = {
     }
   `,
   [Theme.Dark]: css`
-    background-color: ${palette.gray.light2};
+    background-color: ${toastBGColor[Theme.Dark]};
```

Comment:
I see `toastBGColor` is only being referenced here. The colors could just be here instead of being in a separate object

Reply:
nvm, I see it's being referenced from other styles files.